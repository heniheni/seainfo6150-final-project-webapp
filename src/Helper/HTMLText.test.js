import React from "react";
import HTMLTest from "./HTMLText";


describe("HTMLText tests", () => {

    it("renders correctly", () => {
      const { container } = render(<HTMLTest text="dummytext" />);
      expect(container).toMatchSnapshot();
    });
  });