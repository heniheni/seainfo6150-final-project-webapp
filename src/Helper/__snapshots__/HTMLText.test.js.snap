// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HTMLText tests renders correctly 1`] = `
<div>
  <div
    class="detailedTextStyling"
  >
    dummytext
  </div>
</div>
`;
